README.md

```diff
@@ -1,2 +1,33 @@
-# mv-oop.rb
-An educational project to practice OOP in Ruby
+# School Library Exercise
+
+This is an educational project to practice OOP in Ruby
+
+## Built with
+
+This project is made using Ruby language.
+
```

Comment:
- [ ] As the previous reviewer requested, kindly provide steps to setup the project. Imagine that someone viewing  your project has no idea how to clone and start using the app, there are so many languages and frameworks, ruby is only one of them.. I suggest to use this as a template: 
```

## Getting Started

To get a local copy up and running follow these simple example steps.

- copy this link `https://github.com/ahangarha/mv-oop.rb.git`.
- get the directory that you want to clone the repository.
- open the command prompt in this directory.
- write `https://github.com/ahangarha/mv-oop.rb.git.`
- go to the repository folder in your command prompt.
- run `bundle install`.
- run `npm install`
- create database in your postgre called `Blog_development`
- run `rails db:migrate`
- To run the test run `rspec` in your command prompt.
- To run the rails server run  `rails s` in your command prompt.

```

> This template is from Rails project, feel free to customize some steps 😉

Reply:
@shoiraTa I have never mentioned in any of my past projects how to clone the project. If I want to do that, I have to say how to install git as well. If someone is seeing my project, should be able to get the project. For that, he doesn't need to clone. Can simply download the project as a zip file.

There is no requirement for setting up the project. We only have three ruby files containing classes.

I asked him and he didn't reply. May you tell me how to setup this project? Even the person doesn't need to run `bundle install` since that is needed only for linter check. We have no setup, no database, nothing. I can imagine we need something like this in later stage of this project but not now.

Please let me know what to put as setup steps.

Reply:
Hi @ahangarha, I understand your frustration. But every project should have a section in README describing how to setup the project. 
Regarding my review, I was following the guidelines on providing review. If something requested by the previous reviewer is not implemented, the PR cannot be approved. 
I am sad to know that you are so unhappy with review.  If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Reply:
I am peaceful and have nothing against you :white_flower: I just shared my concern.

Last week, I got approval for my database project while later I found three critical problems in my queries. I don't want that kind of approval for sure. I understand you are try to do your best but really for this project there is no setup at this stage. We only have three basic files containing classes. I know we must have something for the last PR at the end of the week but there is nothing to setup now. That is what I think.

Cloning is not setup. One can download it as a zip file and do the rest. Not bundle install also is needed at this stage. I might be wrong. I would love to get constructive feedback in this regard.